roles/postgres/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+postgres_locale: en_US.UTF-8
+postgres_roles: 'NOSUPERUSER,NOCREATEDB,NOCREATEROLE,NOLOGIN'
```

Comment:
from a yaml perspective, may be better to make these a real list and we can use jinja to convert them to a CSV